pkg/apis/serving/v1/revision_validation.go

```diff
@@ -102,7 +102,7 @@ func (current *RevisionTemplateSpec) VerifyNameChange(ctx context.Context, og Re
 	} else if diff != "" {
 		return &apis.FieldError{
 			Message: "Saw the following changes without a name change (-old +new)",
-			Paths:   []string{apis.CurrentField},
+			Paths:   []string{"metadata.name"},
```

Comment:
this is not really correct path, I think, since in service validation this will be "service.spec.template.metadata.name" and service validation is not really setup for the prefix.

Reply:
Currently error display like below
`Saw the following changes without a name change (-old +new): spec.template`

If we add `metadata.name`
It will display the full path
``Saw the following changes without a name change (-old +new): `spec.template.metadata.name`

do you mean `spec.template.metadata.name` is not correct?

Reply:
No that sounds reasonable. Cool. I didn't see where service appended the prefix, but it might be even further apart.